spec/controllers/your_details_controller_spec.rb

```diff
@@ -100,6 +102,63 @@ module Wpcc
       end
     end
 
+    describe 'sessions' do
+      after { Timecop.return }
+
+      it "'wpcc_expires_at' key should always be updated" do
+        session[:wpcc_expires_at] = 5.minutes.ago
+        get :new, nil, some_session
+
+        expect(session[:wpcc_expires_at]).to be > 5.minutes.ago
+      end
+
+      context 'when session has expired' do
+        before do
+          wpcc_expires_at = 5.minutes.ago
+
+          Timecop.travel(Time.current + 30.minutes)
+
+          get :new,
+              nil,
+              age: 34,
+              gender: 'male',
```

Comment:
I made some changes, see what you think. The difficulty I'm having is comparing the real session before and after an action in order to properly test the session expiry time. If I send a 'fake' session along with the action, I'm not sure I'm actually testing that the session details change. 

The difference is that in the create test, I'm comparing the session with what gets posted as opposed to comparing the session before the action and the session after the action.

Maybe I'm making this more difficult than needs be?